test/Swashbuckle.AspNetCore.SwaggerGen.Test/Swashbuckle.AspNetCore.SwaggerGen.Test.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.5.3" />
```

Comment:
I've added [FluentAssertions package](https://www.nuget.org/packages/FluentAssertions/) to easily compare OpenApi objects. Another way was to write very complicated code to check if OpenApi objects are equivalent.